assemble-and-restart

```diff
@@ -13,6 +14,13 @@ else
     /usr/libexec/s2i/assemble
 fi
 
+###
+# This is only to solve wildfly issue 
+#Ref: https://github.com/redhat-developer/odo/issues/445
+mkdir  /opt/app-root/backup
```

Comment:
Just to keep the convention, can we use `mkdir -p` here. ( e.g. https://github.com/redhat-developer/odo-supervisord-image/blob/master/s2i-setup#L9)

Reply:
yeah we can, but `/opt/app-root` will always be there, that's why I didn't added `-p`